cli/index.ts

```diff
@@ -328,19 +357,22 @@ export function runCli() {
             }
           },
           schemaSuffixOverrideOption,
-          jsonOutputOption
+          jsonOutputOption,
+          varsOption
```

Comment:
you haven't added this to the list of options for `run` and `test`

Reply:
How come? It'd be weird if running `dataform compile` worked, but `dataform run` didn't because you couldn't inject a variable where it was required. It might also be useful for people wanting to reference a different dataset during testing rather than compilation, for example

Reply:
I think you misread/misunderstood my comment? I'm just saying that you haven't added this option to `run` or `test`, even though you depend upon it in those commands.

Reply:
Yup lol